utility/mirrormanager2.spec

```diff
@@ -255,28 +251,28 @@ install -m 644 utility/mirrormanager2.cfg.sample \
 
 # Install crawler logrotate definition
 install -m 644 utility/mm2_crawler.logrotate \
-    $RPM_BUILD_ROOT/%{_sysconfdir}/logrotate.d/mm2_crawler
+    $RPM_BUILD_ROOT/%{_sysconfdir}/logrotate.d/mirrormanager2-crawler
 
 # Install umdl logrotate definition
-install -m 644 utility/mm2_umdl.logrotate \
-    $RPM_BUILD_ROOT/%{_sysconfdir}/logrotate.d/mm2_umdl
+install -m 755 utility/mm2_umdl.logrotate \
+    $RPM_BUILD_ROOT/%{_sysconfdir}/logrotate.d/mirrormanager2-backend
 
 # Install WSGI file
 install -m 644 utility/mirrormanager2.wsgi \
     $RPM_BUILD_ROOT/%{_datadir}/mirrormanager2/mirrormanager2.wsgi
-install -m 644 mirrorlist/mirrorlist_client.wsgi \
+install -m 755 mirrorlist/mirrorlist_client.wsgi \
```

Comment:
Not sure 755 is necessary for a wsgi script.

Reply:
This was another error from rpmlint, I think it was `non-executable-script` Changing the permissions to 755 solved it.

Reply:
This is never directly executed, so we should keep it at 644.

Reply:
Adding `non-executable-script` error to rpmlintrc file.